src/briefcase/platforms/macOS/__init__.py

```diff
@@ -745,11 +739,9 @@ def package_app(
                 )
                 self.notarize(self.binary_path(app), team_id=team_id)
 
-            with self.input.wait_bar(
-                f"Archiving {self.distribution_path(app).name}..."
-            ):
+            with self.input.wait_bar(f"Archiving {dist_path.name}..."):
```

Comment:
To clarify since the diff looks like more may have changed than it did....I just added a Wait Bar here to better align with `zip` packaging and because I initially thought Briefcase froze when it got here.